kore/src/Kore/Internal/SideCondition.hs

```diff
@@ -3,6 +3,9 @@ Copyright   : (c) Runtime Verification, 2020
 License     : NCSA
 -}
 
+-- For instance Applicative:
```

Comment:
I am curious why this extension is needed now, but this comment does not help me. In particular, I didn't find any Applicative instance in this file.

Reply:
Yes, it doesn't seem to be used. I wonder what it is left-over from...